lib/util/cache/repository/index.ts

```diff
@@ -6,70 +7,99 @@ import type {
   RepositoryCacheConfig,
 } from '../../../config/types';
 import { logger } from '../../../logger';
-import type { Cache } from './types';
+import type { RepoCache, RepoCacheData } from './types';
 
 // Increment this whenever there could be incompatibilities between old and new cache structure
-export const CACHE_REVISION = 10;
+const CACHE_REVISION = 11;
```

Comment:
Can you keep it the same if you include the migration logic?

Reply:
BTW I can live with bumping the revision if we really need to, but was just hoping to avoid it.

Reply:
I think bump is more logically correct, since the structure has changed in the non-compatible way. Though, these changes will be useful for further repo cache work.

Reply:
ok, then keep the bump